sre/roles/faults/tasks/inject_custom.yaml

```
@@ -1,4 +1,13 @@
 ---
+- name: Low Mem CPU Constraints
+  ansible.builtin.import_tasks:
+    file: inject_low_mem_cpu_constraints.yaml
+  vars:
+    spec: "{{ fault.custom.misconfigured_service_port }}"
+  when:
+    - fault.custom.name == 'low-mem-cpu-constraints'
```

Maybe let's change the name to this one. Should be more in line with the suggestions proposed by @rohanarora .

```suggestion
    - fault.custom.name == 'low_resource_limits'
```